covid19-dashboard/app.yaml

```diff
@@ -0,0 +1,8 @@
+runtime: python37
+instance_class: B8
+env_variables:
+   _ENV_VAR_API_KEY: "AIzaSyCNCHj4uelGpDzcnU-o2cZD44Hs9J78n8I"
```

Comment:
We don't want to reveal this in github code. Can you try using flask config: https://explore-flask.readthedocs.io/en/latest/configuration.html#instance-folder

Reply:
fixed